test_framework/terraform/aws/centos/main.tf

```diff
@@ -261,7 +261,7 @@ resource "aws_instance" "lh_aws_instance_controlplane" {
   }
 
   key_name = aws_key_pair.lh_aws_pair_key.key_name
-  user_data = var.arch == "arm64" ? data.template_file.provision_arm64_server.rendered : data.template_file.provision_amd64.rendered
+  user_data = var.k8s_distro_name == "k3s" ? data.template_file.provision_k3s_server.rendered : data.template_file.provision_rke.rendered
```

Comment:
This ternary expression will be adjusted when supporting rke2 in near future.

Reply:
Yes, this will be modified when adding RKE2 support.